ansible/roles/atmo-ephemeral-mount/tasks/main.yml

```diff
@@ -48,6 +48,14 @@
     copy:
       src: 'DATA-LOSS-WARNING.txt'
       dest: '{{ MOUNT_PATH }}/DATA-LOSS-WARNING.txt'
+
+  - name: Change directory ownership
+    file:
+      state: directory
+      path: '{{ MOUNT_PATH }}'
+      recurse: yes
+      owner: '{{ ATMOUSERNAME }}'
+      group: "{{ ATMO_USER_GROUP }}"
```

Comment:
We don't want to change permissions of every file in their scratch directory, this can take a long time and lock up the file system if they have many files. We should just not recurse here.

Reply:
FIxed